pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -124,3 +130,12 @@ func fixGoodhostsBinaryCached() error {
 	logging.Debug("goodhost binary cached")
 	return setSuid(goodhostPath)
 }
+
+func getCurrentOcversion() (string, error) {
+	ocPath := filepath.Join(constants.CrcOcBinDir, constants.OcBinaryName)
+	stdOut, _, err := crcos.RunWithDefaultLocale(ocPath, "version", "--client")
```

Comment:
> wondering if the parsing could be simpler/more robust with -oyaml or -ojson? 

yes we can use the json or yaml output and put into the struct to get just the version info but even with current scenario it is not that bad.

> On the other hand, this makes it very similar to getCurrentLibvirtDriverVersion() introduced later which could have its use.

Both have different argument to get the version, and that was initially the point to pass function for version info.

Reply:
> Both have different argument to get the version, and that was initially the point to pass function for version info.

Yup, only thinking out loud that maybe it will be useful that they are similar so that we can use something like `func() { return genericGetVersion("version", "--client") }` as the `getVersion` method.